l10n_ch_qr_bill_scan/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Switzerland - QR-bill scan",
+    "summary": "Scan QR-bill to create vendor bills",
+    "version": "10.0.1.0.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "category": "Localization",
+    "website": "https://github.com/OCA/l10n-switzerland",
+    "license": "AGPL-3",
+    "depends": ["l10n_ch", "account_invoice_import"],
```

Comment:
we need python-dependency to have complaining on install
pyzbar==0.1.8
numpy==1.16.2
pdf2image==1.13.1
opencv-python==4.2.0.34